addons/account/models/company.py

```diff
@@ -164,7 +164,7 @@ class ResCompany(models.Model):
             'padding': 5,
             'use_date_range': True,
             'company_id': self.id,
-            'prefix': 'BATCH/%(year)s/',
+            'prefix': 'PAY/%(year)s/',
```

Comment:
This change is good but i think that if runbot was working like a normal pr, you would have a lot of test failing 👀 Take a look at addons/account/tests/test_account_payment_register.py 😄 Run the tests and check the ones that fails and fix them 😄